tests/ports/esp32/ssl_dtls.py

```diff
@@ -0,0 +1,17 @@
+# Very basic test of ssl.SSLContext DTLS.
```

Comment:
There only needs to be one common test for all ports.  Please move this test to `tests/extmod/tls_dtls.py`, and then remove the unix specific one.

Reply:
Done!